### PR TITLE
Add release note for IPsec north-south traffic encryption

### PR DESCRIPTION
**Pull request by jab-rh**
- https://issues.redhat.com/browse/OSDOCS-6862

Version(s): 4.14

Issue:

Link to docs preview: https://64835--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-networking-ovn-kubernetes-ipsec-support-for-external-traffic

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jab-rh** [comment]
@yuvalk, can you take a look at this draft release note when you have a minute? Thanks!

**Comment by anuragthehatter** [comment]
Following bug are open

https://issues.redhat.com/browse/RHELPLAN-166178

https://issues.redhat.com/browse/RHEL-12278

https://issues.redhat.com/browse/OCPBUGS-19817

**Comment by JoeAldinger** [comment]
@jab-rh https://github.com/openshift/openshift-docs/pull/65876/files

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -222,6 +222,15 @@ With this release, you can access the NMstate Operator and resources such as the
 
 IPsec is now supported on the IBM Cloud platform for clusters that use the OVN-Kubernetes network plugin, which is the default in {product-title} 4.14. For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption].
 
+[id="ocp-4-14-networking-ovn-kubernetes-ipsec-support-for-external-traffic"]
+==== OVN-Kubernetes network plugin support for IPsec encryption of external traffic
+
+IPsec now supports encryption of external traffic, also known as north-south traffic. IPsec already supports encryption of network traffic between pods, known as east-west traffic.
```
Comment (yuvalk) [inline release_notes/ocp-4-14-release-notes.adoc:228]: ~~IPsec~~ Openshift now supports IPsec encryption of external traffic, ...

Comment (yuvalk) [inline release_notes/ocp-4-14-release-notes.adoc:228]: the two can operate togther/in conjunction to provide full in-transit encryption solution for Openshift clusters

**Comment by jab-rh** [comment]
@yuvalk, updated, thanks!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -222,6 +222,15 @@ With this release, you can access the NMstate Operator and resources such as the
...
 IPsec is now supported on the IBM Cloud platform for clusters that use the OVN-Kubernetes network plugin, which is the default in {product-title} 4.14. For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption].
 
+[id="ocp-4-14-networking-ovn-kubernetes-ipsec-support-for-external-traffic"]
+==== OVN-Kubernetes network plugin support for IPsec encryption of external traffic
+
+{product-title} now supports encryption of external traffic, also known as north-south traffic. IPsec already supports encryption of network traffic between pods, known as east-west traffic. You can use both features in conjunction to provide full in-transit encryption for {product-title} clusters.
+
+To use this feature, you need to define an IPsec configuration tuned for your network infrastructure. For more information, refer to
+// TODO - enable
+// xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#nw-ovn-ipsec-north-south-enable_configuring-ipsec-ovn[Enabling IPsec encryption for external IPsec endpoints].
```
Comment (JoeAldinger) [inline release_notes/ocp-4-14-release-notes.adoc:232]: I'm not sure that anchor will work `#nw-ovn-ipsec-enable_configuring-ipsec-ovn`. I'm guessing this is tied to 4.14 updates. If so, disregard my comment.

**Review by JoeAldinger — COMMENTED** [review]
One small question/suggestion

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (jab-rh) [reply release_notes/ocp-4-14-release-notes.adoc:232]: @JoeAldinger, possible I'll have to fix it; I'll enable it once the content PR is merged. Thanks!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -222,6 +222,15 @@ With this release, you can access the NMstate Operator and resources such as the
 
 IPsec is now supported on the IBM Cloud platform for clusters that use the OVN-Kubernetes network plugin, which is the default in {product-title} 4.14. For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption].
 
+[id="ocp-4-14-networking-ovn-kubernetes-ipsec-support-for-external-traffic"]
+==== OVN-Kubernetes network plugin support for IPsec encryption of external traffic
+
+{product-title} now supports encryption of external traffic, also known as north-south traffic. IPsec already supports encryption of network traffic between pods, known as east-west traffic. You can use both features in conjunction to provide full in-transit encryption for {product-title} clusters.
```
Comment (anuragthehatter) [inline release_notes/ocp-4-14-release-notes.adoc:228]: `IPSec already supports encryption of network traffic between nodes` ? cc @yuvalk

**Comment by jab-rh** [comment]
@yuvalk @anuragthehatter 

Merge freeze is Friday EOD eastern time; Can you take a look? Then I can finalize and get this into the docs for the 4.14 release. Thanks!

**Comment by jab-rh** [comment]
@anuragthehatter, looks like this is actually tech preview? I've added it to the tech preview table. Can you confirm? Thanks!

**Pull request merged**